log2asc.c

```diff
@@ -407,7 +407,22 @@ int main(int argc, char **argv)
 			}
 		}
 		tv.tv_sec = sec;
-		tv.tv_usec = usec;
+
+		/*
+		 * ensure the fractions of seconds are 6 or 9 decimal places long to catch
+		 * 3rd party or handcrafted logfiles that treat the timestamp as float
```

Comment:
I would prefer a comment that talks about nanoseconds instead of float.

Reply:
I kept this comment the same as in canplayer.c except for adding "or 9"